torch/csrc/jit/codegen/cuda/ir_base_nodes.h

```diff
@@ -70,6 +71,14 @@ class ExprPasskey {
 
 TORCH_CUDA_CU_API void swap(Fusion& a, Fusion& b) noexcept;
 
+#define DECLARE_CLONE \
+  virtual Statement* clone(IrCloner* ir_cloner) const override;
+
+#define DEFINE_CLONE(ClassName)                            \
+  Statement* ClassName::clone(IrCloner* ir_cloner) const { \
+    return IrBuilder::clone(this, ir_cloner);              \
+  }
```

Comment:
Can we undef these somewhere? Otherwise, we should use names like `PYTORCH_NVFUSER_DECLARE_CLONE`.

Reply:
I don't think so. These macros will be used in cpp files. I will rename it as `NVFUSER_DECLARE_CLONE`.

Reply:
Renamed